src/index.pug

```diff
@@ -0,0 +1,22 @@
+doctype 
+html
+  head
+    title adding some content for the website
+    link(rel="stylesheet", href = "style.css")
+  body 
+    .content 
+        .section-title  Welcome to My CV 
```

Comment:
too much indentation, `.section-title` should be the child of `.content`at the same level as `button`